src/CodeGen_PTX_Dev.cpp

```diff
@@ -742,7 +756,10 @@ bool CodeGen_PTX_Dev::supports_atomic_add(const Type &t) const {
     }
     if (t.is_float() && t.bits() == 64) {
         // double atomics are supported since CC6.1
-        return target.has_feature(Target::CUDACapability61);
+        return (target.has_feature(Target::CUDACapability61) ||
```

Comment:
At this point, a helper function of `has_cuda_capability_at_least(num)` seems like it would pay for itself

Reply:
EDIT: or, alternately, use `get_cuda_capability_lower_bound() >= 61` here instead